token-lending/program/src/state/reserve.rs

```diff
@@ -290,10 +291,24 @@ impl Reserve {
             }
         }
 
+        // calc the bonus amount
+        let base_amount = Decimal::from(withdraw_amount).try_div(bonus_rate)?;
+        let bonus_amount = Decimal::from(withdraw_amount).try_sub(base_amount)?;
+
+        // protocol takes a percentage of the bonus
+        let mut protocol_fee_amount = bonus_amount.try_mul(protocol_fee_rate)?.try_floor_u64()?;
+        // minimum protocol fee
+        if protocol_fee_amount == 0 {
+            protocol_fee_amount = 1;
```

Comment:
Doesn't enforcing a minimum protocol fee amount completely disincentive liquidators from liquidating sufficiently small liquidations?
I know they don't really go after the small ones anyway, but it seems like we might not want to have 0 incentivize liquidation opportunities maybe?
idk

Reply:
this only affects the smallest of liquidations, which no one is incentivized to do anyway.  wont liquidations already fail if withdraw amount is 0? (if you look in processor)